website/docs/components/color-picker/examples/customTrigger.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ColorPicker from '@semcore/color-picker';
+import Input from '@semcore/input';
+import { Box } from '@semcore/flex-box';
+
+const Demo = () => {
+  const [value, setValue] = React.useState('#98848D');
+
+  return (
+    <ColorPicker value={value} onChange={setValue}>
+      <Input ml={1} w={200}>
+        <Input.Addon role="button" interactive>
+          <ColorPicker.Trigger tag={Box}>
+            <div
+              style={{
```

Comment:
Why `ColorPicker.Item` or something like that was not used there?

Reply:
this is just example :) but `ColorPicker.Item` has additional extra styles (like hover, selected and etc) for this and its size 28x28px - it is too large